src/List/ListItem.js

```diff
@@ -38,6 +39,16 @@ export const styleSheet = createStyleSheet('ListItem', (theme) => {
       paddingLeft: 16,
       paddingRight: 16,
     },
+    button: {
```

Comment:
That's not directly linked but we can nest a button inside a `<ListItem />`. We do it with the documentation [drawer nav items](https://github.com/callemall/material-ui/blob/next/docs/site/src/components/AppDrawerNavItem.js#L83-L97).

I think that you made a good call here too.